docs/man/docker-create.1.md

```diff
@@ -132,6 +133,9 @@ IMAGE [COMMAND] [ARG...]
                                When specifying ranges for both, the number of container ports in the range must match the number of host ports in the range. (e.g., `-p 1234-1236:1234-1236/tcp`)
                                (use 'docker port' to see the actual mapping)
 
+**--persist-ports**=*true*|*false*
+   Persist published host ports that were dynamically allocated so that they remain the same even after restarting the container.
```

Comment:
Just a suggestion, but we could simplify/clarify this a little with:
 "Allow dynamically allocated, published ports to persist after restarting a container.